regulations/static/regulations/js/unittests/specs/views/main/diff-view-spec.js

```diff
@@ -0,0 +1,46 @@
+require('../../../setup');
+var sinon = require( 'sinon' );
+
+describe('Diff View:', function () {
+
+  var view, $, DiffView;
+
+  before(function () {
+    $ = require('jquery');
+    DiffView = require('../../../../source/views/main/diff-view');
+    sandbox = sinon.sandbox.create();
+  });
+
+  beforeEach(function(){
+
+    // create a new instance of the view
+    view = new DiffView();
+
+    $('title').text('Comparison of 12 CFR § 1002.4 | eRegulations');
+    document.title = 'Comparison of 12 CFR § 1002.4 | eRegulations';
+  });
+
+  it('should construct a view', function() {
+    expect(view).to.be.defined;
+    expect(document.title).to.be.equal('Comparison of 12 CFR § 1002.4 | eRegulations');
```

Comment:
I was testing that the initial title is set correctly so that it can be updated in the following test. This isn't necessary and could definitely be removed.